tests/test_validate_csvs.py

```diff
@@ -11,76 +11,84 @@ class TestCSVIntegrity:
     This is important for the subsequent matrix
     multiplications done in the model.
     """
+
     @pytest.fixture(scope="class")
     def initial_csv_data(self):
         directory = Path("parameters/")
-        csv_files = [f for f in os.listdir(directory) if
-                     (f.endswith('.csv') and
-                      (f.startswith('lockdown') or
-                       f.startswith('actions')))
-                     ]
+        csv_files = [
+            f
+            for f in os.listdir(directory)
+            if (
+                f.endswith(".csv")
+                and (f.startswith("lockdown") or f.startswith("actions"))
+            )
+        ]
```

Comment:
I think that the pattern that you introduced below would make this part more readable too:

```suggestion
        csv_files = list(directory.glob("lockdown*.csv")) + list(directory.glob("actions*.csv"))
```